cluster/common.sh

```diff
@@ -58,16 +60,17 @@ function create-kubeconfig() {
     )
   fi
   local user_args=()
-  if [[ -z "${KUBE_USER:-}" || -z "${KUBE_PASSWORD:-}" ]]; then
-    user_args+=(
-     "--token=${KUBE_BEARER_TOKEN}"
-    )
-  else
+  if [[ ! -z "${KUBE_USER:-}" && ! -z "${KUBE_PASSWORD:-}" ]]; then
```

Comment:
[Yes](https://github.com/GoogleCloudPlatform/kubernetes/blob/master/pkg/kubectl/cmd/config/create_authinfo.go#L207). You can only set username/password or token, and doing so will clear the other if it is present.